cmd/promtool/tsdb.go

```diff
@@ -456,13 +379,44 @@ func printBlocks(blocks []tsdb.BlockReader, humanReadable *bool) {
 	}
 }
 
-func getFormatedTime(timestamp int64, humanReadable *bool) string {
-	if *humanReadable {
+func getFormatedTime(timestamp int64, humanReadable bool) string {
+	if humanReadable {
 		return time.Unix(timestamp/1000, 0).String()
 	}
 	return strconv.FormatInt(timestamp, 10)
 }
 
+func AnalyzeBlock(path, blockID string, limit int) error {
```

Comment:
instead of this I think it should be like:

func analyzeBlock(b tsdb.BlockReader, limit int) error {
	openBlock(.....


	// the rest of the function opertions.
}

Reply:
Why do we need to change this? We already have AnalyzeBlock(path, blockID string, limit int) and analyzeBlock(b tsdb.BlockReader, limit int). There is some block check logic in AnalyzeBlock so I think it is OK to keep both.

Reply:
`AnalyzeBlock` and `analyzeBlock`  doesn't describe what each function does. Also I don't see any reason why would `AnalyzeBlock` be exposed.

My proposal refactors the function names so that it describes what each function does.

Reply:
Done. Could you review that again?